Modules/@babylonjs/react-native/android/src/main/java/com/babylonreactnative/EngineViewManager.java

```diff
@@ -1,29 +1,39 @@
 package com.babylonreactnative;
 
+import android.widget.FrameLayout;
+
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
 import com.facebook.react.bridge.ReadableArray;
 import com.facebook.react.common.MapBuilder;
 import com.facebook.react.uimanager.SimpleViewManager;
 import com.facebook.react.uimanager.ThemedReactContext;
+import com.facebook.react.uimanager.annotations.ReactProp;
 
 import java.util.Map;
 
 public final class EngineViewManager extends SimpleViewManager<EngineView> {
     public static final int COMMAND_TAKE_SNAPSHOT = 0;
     public static final String COMMAND_TAKE_SNAPSHOT_NAME = "takeSnapshot";
+    private Boolean isTransparent = false;
 
     @NonNull
     @Override
     public String getName() {
         return "EngineView";
     }
 
+    @ReactProp(name = "isTransparent")
+    public void setIsTransparent(EngineView view, Boolean _isTransparent) {
+      isTransparent = _isTransparent;
```

Comment:
Well, strangely it happened for me before the initialization because it was working correctly. But yeah debugging it step by step might show otherwise.